VERSION "1.0"
NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_DEF_
    BA_DEF_DEF_REL_
    BA_DEF_REL_
    BA_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:

BU_: ECU1 ECU2

BO_ 100 ENGINE_STATUS: 8 ECU1
 SG_ EngineSpeed : 0|16@1+ (0.125,0) [0|8000] "RPM" ECU2
 SG_ EngineTemp : 16|8@1+ (1, -40) [-40|215] "C" ECU2

BO_ 200 VEHICLE_SPEED: 8 ECU1
 SG_ VehicleSpeed : 0|16@1+ (0.01,0) [0|250] "km/h" ECU2

BO_ 300 BRAKE_INFO: 8 ECU1
 SG_ BrakePedalStatus : 0|1@1+ (1,0) [0|1] "" ECU2
 SG_ BrakePressure : 8|16@1+ (0.1,0) [0|1000] "bar" ECU2

BO_ 400 STEERING_INFO: 8 ECU1
 SG_ SteeringAngle : 0|16@1- (0.1,0) [-500|500] "deg" ECU2
 SG_ SteeringTorque : 16|16@1- (0.01,0) [-327|327] "Nm" ECU2
